java-base/src/main/java/net/cryptic_game/backend/base/daemon/DaemonRegisterPacket.java

```diff
@@ -43,9 +43,9 @@ public DaemonRegisterPacket(final Channel channel, final String name, final Json
 
     @Override
     public JsonObject serialize() {
-        return JsonBuilder.anJSON()
-                .add("name", this.getName())
-                .add("functions", JsonUtils.toArray(this.functions))
+        return JsonBuilder
+                .create("name", this.getName())
+                .add("functions", JsonUtils.toArray(this.functions, Function::serialize))
```

Comment:
You dont need `JsonUtils#toArray(Collection<?>)`. The new Json Builder can hadle `Collections` without any help.